src/features/SendAssets/SendAssets.reducer.tsx

```diff
@@ -61,6 +62,11 @@ export const sendAssetsReducer = (state: State, action: ReducerAction): State =>
       return { ...state, txConfig };
     }
 
+    case sendAssetsReducer.actionTypes.SET_TXCONFIG: {
```

Comment:
Perhaps we can call this something related to resubmit then?

Reply:
I think `SET_TXCONFIG` is more accurate because it can be used for more than just resubmitting txs. we'll use the same functionality for transaction cancellation.